syntax/blame.sublime-settings

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Several places in the code we have lines like this which we should remove if we go for adding these settings.

Reply:
???? 

A settings file needs to start with `{` as it represents a JSON object. Removing it would break the settings file.

Reply:
https://github.com/divmain/GitSavvy/blob/master/core/commands/blame.py#L57-L59
I lost this link. Sorry

Reply:
Aaahh. Know what you mean.

Reply:
Removed the duplicated settings from the plugins.